packages/logs/lib/es/schema.ts

```diff
@@ -96,7 +97,8 @@ const props: Record<keyof MessageRow | keyof OperationRow, estypes.MappingProper
     updatedAt: { type: 'date' },
     startedAt: { type: 'date' },
     expiresAt: { type: 'date' },
-    endedAt: { type: 'date' }
+    endedAt: { type: 'date' },
+    durationMs: { type: 'integer' }
```

Comment:
cannot duration be calculated by doing `endedAt - startedAt`?

Reply:
it's possible yes, but it's super slow at scale